#!/usr/bin/env python
PACKAGE = "ffcc_catkin"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("Lu_offset",    double_t,    0, "Lu offset", 0,  -2.000, 2.000)
gen.add("Lv_offset",    double_t,    0, "Lv offset", 0,  -2.000, 2.000)
gen.add("uv0",    double_t,    0, "uv0", -1.421875,  -10.0, 10.0)
gen.add("saturation_threshold",  double_t,    0, "Saturation threshold", 1.0,  0.0, 1.0)

exit(gen.generate(PACKAGE, "dynamic_tutorials", "FFCCCatkin"))